src/lib/pages/backoffice/Acquisition/Order/OrderDetails/OrderLines.js

```diff
@@ -110,6 +112,9 @@ export const OrderLine = ({ line, LeftColumn, MiddleColumn, RightColumn }) => {
             publicationYear={line.document.publication_year}
           />
         </Link>
+        <Item.Meta>
+          {renderSubtitle(_get(line, 'document.alternative_titles'))}
```

Comment:
I don't think you need `lodash` here, and same for the occurrences below:
```suggestion
          {renderSubtitle(line.document?.alternative_titles)}
```
Please make sure that you test it well.